src/async-wrap.h

```diff
@@ -71,9 +71,17 @@ namespace node {
 #define NODE_ASYNC_CRYPTO_PROVIDER_TYPES(V)
 #endif  // HAVE_OPENSSL
 
+#if HAVE_INSPECTOR
+#define NODE_ASYNC_INSPECTOR_PROVIDER_TYPES(V)                                \
+  V(INSPECTORJSBINDING)
+#else
+#define NODE_ASYNC_INSPECTOR_PROVIDER_TYPES(V)
+#endif  // HAVE_INSPECTOR
+
 #define NODE_ASYNC_PROVIDER_TYPES(V)                                          \
   NODE_ASYNC_NON_CRYPTO_PROVIDER_TYPES(V)                                     \
-  NODE_ASYNC_CRYPTO_PROVIDER_TYPES(V)
+  NODE_ASYNC_CRYPTO_PROVIDER_TYPES(V)                                         \
+  NODE_ASYNC_INSPECTOR_PROVIDER_TYPES(V)                                      \
```

Comment:
should this end with a `\`?

Reply:
Yes, yes it should. Thanks for noticing.

Reply:
Oh oops, didn't realize it already had a `\`. I'd say it's good practice to always have a trailing backslash to reduce future diffs, but will remove.